<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QMainWindow" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
    <pointsize>10</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLineEdit" name="loginTxt">
      <property name="text">
       <string>PythON</string>
      </property>
      <property name="placeholderText">
       <string>Логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordTxt">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>Simple is better than complex</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="confirmBtn">
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
